### ISSUE TITLE
Ability to set the true and false values in the tickCross editor

### ISSUE THREAD

**Issue by richard-slabbert94** [olifolkerd/tabulator#3186]
**Is your feature request related to a problem? Please describe.**
In my MySQL database, I have a column where true/false values are saved as 1 or 0 respectively.  I have set the column to a tickCross editor in Tabulator.  The column data is loaded as 0's or 1's.  If I change the value via the tickCross editor and then try to get the new value using cell.getValue(), the value has changed from '1' or '0' to 'true' or 'false'.  It would be ideal if it were to stay as an integer, since I then need to then use a mutator to change the true/false back to 0 or 1.

I created a fiddle where if you click on the "name" column it console logs the value of "driver" (which is a 0/1 column).  You will see that after editing the "driver" field, the value logged will change to true/false.
Here is a fiddle you can check: [Fiddle](https://jsfiddle.net/rslabbert/7q6fLos2/9/)

**Describe the solution you'd like**
It might be nice to include a "tickOnValue" and "tickOffValue" value in the settings, as this will allow anyone to save any value which is handy.

For example,

```
{title:"Example", field:"example", editor:"tickCross", editorParams:{
    tristate:true,
    tickOnValue : 1,
    tickOffValue : 0,
    indeterminateValue:"n/a",
    elementAttributes:{
        maxlength:"10", //set the maximum character length of the input element to 10 characters
    }
}}
```

**Describe alternatives you've considered**
As I have said, this can be done using a mutator, but it would be nice if it were handled by Tabulator itself as an extra option.

```
mutator: function(value) {
      var integer = value === true || value == 1 ? 1 : 0;
      return integer;
}
```

**Comment by olifolkerd** [comment]
Hey @richard-slabbert94 

That is an excellent idea, i shall add that to the roadmap for a later release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @richard-slabbert94 

I have pushed an update for this to the 5.2 branch. This will be included in the 5.2 release later this month.

Cheers

Oli :)

**Comment by richard-slabbert94** [comment]
Thanks Oli!

You're a legend.
